### DISCUSSION TITLE
Open links in browser instead of new window

### DISCUSSION THREAD

**Discussion by gilbitron** [NativePHP/laravel#212] [Q&A] [answered]
At the moment, links with `target="_blank"` open in a new window. Is there a way to make links open the default browser instead?

For example, in electron you can do:

```js
require("shell").openExternal("http://www.google.com")
```

**Comment by pb30** [comment, marked as answer]
See https://github.com/NativePHP/laravel/discussions/173#discussioncomment-6721552

`Shell::openExternal($url)`
